.github/workflows/tests.yml

```diff
@@ -19,32 +19,37 @@ jobs:
 
   linux:  # {{{
     runs-on: ubuntu-latest
+    if: true
```

Comment:
What is this (and other similar below) for?

Reply:
It is because, when I want to run tests only a subset of the platforms (for instance debugging a windows problem), I wish to disable the other platform running. It can be done adding `if: false` to the other platforms... but only after I make the regular mistake of adding `when: false` and being scolded by github :smile: 

So it's a reminder about the syntax to use to disable a branch.